reference/7.5/Microsoft.PowerShell.Diagnostics/Get-WinEvent.md

```diff
@@ -500,7 +500,7 @@ from newest to oldest.
 
 Event Tracing for Windows (ETW) writes events to the log as events occur. The events are stored in
 the order of oldest to newest. An archived ETW file is saved as an `.etl` such as **TraceLog.etl**.
-The events are listed in the order in which they are written to the log, so the *Oldest* parameter
+The events are listed in the order in which they are written to the log, so the _Oldest_ parameter
```

Comment:
Parameter names (in prose) should be bold - see https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/editorial-checklist?view=powershell-7.4#formatting
```suggestion
The events are listed in the order in which they are written to the log, so the **Oldest** parameter
```

Reply:
Ok, so the original was wrong in this case. From MD perspective, it just pointed to the wrong character being used.